app/ide-desktop/lib/content/package.json

```diff
@@ -17,8 +17,8 @@
   "scripts": {
     "typecheck": "npx tsc --noEmit",
     "lint": "npx --yes eslint src",
-    "build": "cd ../dashboard/src/authentication && npx tailwindcss -i src/styles/index.css -o ../../../../../../dist/gui/assets/tailwind.css -c tailwind.config.ts --minify && cd ../../../content && ts-node bundle.ts",
-    "watch": "cd ../dashboard/src/authentication && npx tailwindcss -i src/styles/index.css -o ../../../../../../dist/gui/assets/tailwind.css -c tailwind.config.ts --minify && cd ../../../content && ts-node watch.ts",
+    "build": "ts-node bundle.ts",
+    "watch": "ts-node watch.ts",
     "start": "ts-node start.ts"
```

Comment:
Does it mean that tailwind CSS will be regenerated on every watch command? If so, this is bad, because we want watch to be as fast as possible, especially when chaining rust code.

Reply:
actually, i have no clue. build times average 200ms for me when editing css/html

Reply:
wait...

Reply:
> actually, i have no clue. build times average 200ms for me when editing css/html

This would not be a problem if this was a simple build script. But 200ms here and there can cause big problems for our dev speed. So we take a lot of care about it. Thanks for fixing it! Running it once seems like perfect solution.